### PR TITLE
UX Tiers screen

### PR DESCRIPTION
**Pull request by prakashsvmx**
UX Tiers screen

Cosmetic changes for consistency

### CODE REVIEW

**Review by dvaldivia — DISMISSED** [review]
Tested. LGTM

File: portal-ui/src/screens/Console/Common/FormComponents/FileSelector/FileSelector.tsx [hunk 1]
```diff
@@ -112,9 +112,10 @@ const FileSelector = ({
       <Grid
         item
         xs={12}
-        className={`${classes.fieldBottom} ${classes.fieldContainer} ${
-          error !== "" ? classes.errorInField : ""
-        }`}
+        classes={classes}
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Common/FormComponents/FileSelector/FileSelector.tsx:115]: Please remove this
```suggestion
```

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Common/FormComponents/FileSelector/FileSelector.tsx (hunk 1 above)
Comment (bexsoft) [reply portal-ui/src/screens/Console/Common/FormComponents/FileSelector/FileSelector.tsx:115]: Beside of that LGTM

Comment (prakashsvmx) [reply portal-ui/src/screens/Console/Common/FormComponents/FileSelector/FileSelector.tsx]: Thank you. I have updated.

**Review by bexsoft — APPROVED** [review]
LGTM

**Review by dvaldivia — APPROVED** [review]

**Pull request merged**